apps/mobile-alphabet/src/app/Components/Menu-detail.presenter.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Image, Text, View } from 'react-native';
+import config from './Config.json';
+import { AlphabetData } from './Menu';
+
+export function MenuDetialScreen() {
+    const [alphabetData, setAlphabetData] = useState<AlphabetData | null>(null);
+
+    const [selectedLetterSequenceNumber, setSelectedLetterSequenceNumber] = useState<number>(1);
+
+    useEffect(() => {
+        //TODO cache this after fetching it once
+        const fetchData = async () => {
+            try {
+                //TODO use context api
+                const response = await fetch(config.apiUrl, {
+                    mode: 'cors',
+                });
+                setAlphabetData(await response.json());
+            } catch (error) {
+                console.error('Error fetching data:', error);
+            }
+        };
+
+        fetchData();
+    }, []);
+
+    if (!alphabetData) {
+        return <Text>Loading...</Text>;
+    }
+
+    const {
+        data: { alphabet_cards: alphabetCards },
+    } = alphabetData;
+
+    const selectedCard = alphabetCards.find(({ sequence_number: sequenceNumber }) => {
+        return Number.parseInt(sequenceNumber) === selectedLetterSequenceNumber;
+    });
+
+    const {
+        word,
+        letter,
+        sequence_number,
+        card_image,
+        letter_audio,
+        word_audio,
+        standalone_image,
+    } = selectedCard;
+
+    return (
+        <View testID="Detail">
+            <Image
+                style={{ width: 200, height: 200 }}
+                resizeMode="cover"
+                source={{
+                    uri: standalone_image,
+                }}
+            />
+            <Text>Letter: {letter}</Text>
+            <Text>Word: {word}</Text>
+            <Text testID={sequence_number}>Sequence #: {sequence_number}</Text>
+            <Text>Image: {card_image}</Text>
```

Comment:
Not sure what this looks like when spun up, but you may not want the "Image:" label here, just the image itself.  And maybe it goes up top?  Ignore this if it doesn't make sense in the view of the app.